ui/src/runtime/components/Conditional.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <Bake
+    v-if="data"
+    :name="`${path}/${component.type}`"
+    :descriptor="component"
+  />
```

Comment:
now that it is an independent component, let's add `AwaitLoading` in case we use it in other places than data table